client/embed.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+  var wecoWorkEmbeds = document.querySelectorAll('.weco-work-embed');
+  wecoWorkEmbeds.forEach(el => {
+    if (!el.getAttribute('data-weco-enhanced')) {
+      var workId = el.getAttribute('data-weco-work-id');
+      var parentEl = el.parentElement;
+      el.setAttribute('data-weco-enhanced', 'true');
+      el.addEventListener('click', function() {
```

Comment:
…and why not an arrow function?

Reply:
In browser - although it is being compiled.